cime_config/config_grids.xml

```diff
@@ -5402,10 +5496,15 @@
     </gridmap>
 
     <gridmap ocn_grid="RRSwISC6to18E3r5" rof_grid="r05">
-      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r05_to_RRSwISC6to18E3r5.cstmnn.r250e1250_58NS.20240328.nc</map>
+      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r05_to_RRSwISC6to18E3r5_cstmnn.r250e1250_58NS.20240328.nc</map>
```

Comment:
@xylar -- any idea why this line is showing up as changed? Blank space?

Reply:
No, I don't believe it's blank space.  It looks like GitHub is glitching.

Reply:
Ah, no, there's a period that got changed to an underscore.